images/wkdev_sdk/jhbuild/webkit-sdk-deps.modules

```diff
@@ -126,15 +126,14 @@
             version="1.4.2" />
   </meson>
 
-  <meson id="gst-plugins-rs" mesonargs="-Ddoc=disabled -Dexamples=disabled -Dtests=disabled">
+  <meson id="gst-plugins-rs" mesonargs="-Ddoc=disabled -Dexamples=disabled -Dtests=disabled -Daws=disabled -Dwebrtc=disabled -Dwebrtchttp=disabled">
```

Comment:
Why is WebRTC support disabled? We have WebKit bots building with gst-webrtc, and the bots are to be migrated from the Flatpak SDK to the container SDK... so I imagine we would need to keep this enabled?

Reply:
No, that webrtc plugin ships the webrtcsrc and sink elements, which we do not use in WebKit.

Reply:
Ah, understood.  Thanks for the explanation.